README.md

```diff
@@ -179,6 +172,12 @@ On Linux you can see the output of the application in Vs Code itself. Add `gdbtt
 ```
 ![GdbTTY](gdbttyvscode.png)
 
+You can also use these options with `gdbtty`: `xterm`, `gnome-terminal`, `konsole` and `xfce4-terminal`.
+
+### Roadmap
```

Comment:
The roadmap section was explicitly removed by Emilien in https://github.com/OCamlPro/superbol-vscode-debug/pull/1. Do we want it back ?

Reply:
Thanks for noticing!